pcs_test/tier0/common/reports/test_messages.py

```diff
@@ -369,22 +369,50 @@ def test_cannot_be_empty_and_forbidden_characters(self):
 
 
 class DeprecatedOption(NameBuildTest):
-    def test_no_desc_hint_array(self):
+    def test_no_desc_multiple_replacements(self):
         self.assert_message_from_report(
-            "option 'option name' is deprecated and should not be used,"
-            " use 'new_a', 'new_b' instead",
+            "option 'option name' is deprecated and should not be used, "
+            "use 'new_a', 'new_b' instead",
             reports.DeprecatedOption("option name", ["new_b", "new_a"], ""),
         )
 
-    def test_desc_hint_string(self):
+    def test_desc_hint_single_replacement(self):
         self.assert_message_from_report(
-            "option type option 'option name' is deprecated and should not be"
-            " used, use 'new option' instead",
+            "option type option 'option name' is deprecated and should not be "
+            "used, use 'new option' instead",
             reports.DeprecatedOption(
                 "option name", ["new option"], "option type"
             ),
         )
 
+    def test_to_be_removed_no_replacement(self):
+        self.assert_message_from_report(
+            "option 'option name' is deprecated and will be removed",
+            reports.DeprecatedOption("option name", [], "", True),
+        )
+
+    def test_desc_component_removed_no_replacement(self):
+        self.assert_message_from_report(
+            "option type option 'option name' is deprecated by pacemaker and "
+            "will be removed",
```

Comment:
Looking at this message, I'm not sure if users will know what is 'pacemaker' in this context. Maybe we should be more specific, like 'pacemaker package', 'pacemaker project' or something similar which will be more explanatory for the users.